x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -209,7 +209,7 @@ public void testGroupByOrderByScalarOverNonGrouped_WithHaving() {
     }
 
     public void testGroupByHavingNonGrouped() {
-        assertEquals("1:48: Cannot filter by non-grouped column [int], expected [text]",
+        assertEquals("1:48: Cannot filter HAVING on non-aggregate [int]; consider using WHERE instead",
```

Comment:
Why are we losing here the `text` proposal? Can we still have it?

Reply:
Thx for catching it, I had it in mind but forgot to open a relevant issue for that.
Here it is: #36693